lib/bundle/installer.rb

```diff
@@ -45,7 +51,14 @@ def install(entries)
         puts Formatter.error("Homebrew Bundle failed! #{failure} Brewfile #{Bundle::Dsl.pluralize_dependency(failure)} failed to install.")
       end
 
-      failure.zero?
+      unless errored_entries.empty?
+        puts Formatter.error("Homebrew Bundle encountered some errors! #{errored_entries.size} Brewfile #{Bundle::Dsl.pluralize_dependency(errored_entries.size)} failed badly:")
```

Comment:
This is a very long line (both in terms of the code and the output). Could both be tweaked?